src/ActionsImporter/Commands/AzureDevOps/Audit.cs

```diff
@@ -13,6 +13,12 @@ public Audit(string[] args)
     protected override string Name => "azure-devops";
     protected override string Description => "An audit will output a list of data used in an Azure DevOps instance.";
 
+    private static readonly Option<FileInfo> ConfigFilePath = new("--config-file-path")
```

Comment:
Should this be added to the `Options`?

Reply:
woff had it in Valet but not here! good catch